File: src/main/java/by/intexsoft/helloworldtomcat/model/Picture.java
package by.intexsoft.helloworldtomcat.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import javax.persistence.Transient;

/**
 * Picture entity class
 */
@Entity
@Table(name="picture")
public class Picture extends Base {

    /**
     * Field stores picture author
     */
    @Column
    public String author;

    /**
     * Field stores picture name
     */
    @Column
    public String name;

    /**
     * Field stores picture description
     */
    @Column
    public String description;

    /**
     * Field stores picture owner
     */
    @Column
    public String owner;

    /**
     * Field stores picture tags
     */
    @Column
    public String tags;

    /**
     * Field stores image base64 data
     */
    @Column
    public String image;
}

File: frontend/src/app/model/picture.ts
export class Picture {
    id: number;
    author: string;
    name: string;
    description: string;
    owner: string;
    tags: string;
    image: string;

    constructor(id: number, author: string, name: string, description: string, owner: string, tags: string, image: string) {
        this.id = id;
        this.author = author;
        this.name = name;
        this.description = description;
        this.owner = owner;
        this.tags = tags;
        this.image = image;
    }
}